plugins/aver-hid/fu-aver-hid-device.c

```diff
@@ -290,6 +299,11 @@ fu_aver_hid_device_wait_for_reboot_cb(FuDevice *device, gpointer user_data, GErr
 				fu_struct_aver_hid_res_isp_status_get_status(res)));
 		return FALSE;
 	}
+
+	/* send ISP_REBOOT, no response expected */
+	fu_struct_aver_hid_req_isp_set_custom_isp_cmd(req, FU_AVER_HID_CUSTOM_ISP_CMD_ISP_REBOOT);
```

Comment:
so is this needed for just the CAM520Pro3? What happens if you send it to Fone540 for example?

Reply:
Fone540 also works. Fone540 can auto-reboot, but CAM520Pro3 will wait to reboot until it receives the ISP_REBOOT.

Reply:
I'm confused why ISP_REBOOT is being requested in the retry callback 300 times. Do you want to call ISP_REBOOT when the fu_aver_hid_device_write_firmware request has completed just before the call to set FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG?

Reply:
@PierceWangAVer can you pull wip/aver/cam520pro3 and see if that works?

Reply:
It's works. Thank you. We will modify on wip/aver/cam520pro3 and create a new pull request for this feature.